packages/block-editor/src/components/list-view/index.js

```diff
@@ -122,21 +120,25 @@ export default function ListView( {
 	);
 
 	return (
-		<TreeGrid
-			className="block-editor-list-view-tree"
-			aria-label={ __( 'Block navigation structure' ) }
-			ref={ treeGridRef }
-			onCollapseRow={ collapseRow }
-			onExpandRow={ expandRow }
-		>
-			<ListViewContext.Provider value={ contextValue }>
+		<ListViewContext.Provider value={ contextValue }>
+			<ListViewDropIndicator
+				listViewRef={ elementRef }
+				blockDropTarget={ blockDropTarget }
```

Comment:
TODO - try making List View use the Insertion Point state so that the indicators appear in both the main and list view block lists.

Reply:
Tried this out, but it does slow things down a bit, will push my explorations to a separate PR.